docs/source/migration.mdx

```diff
@@ -1611,9 +1618,10 @@ new ApolloServer({
 ## Renamed packages
 
 The following packages have been renamed in Apollo Server 4:
- * `apollo-server-plugin-response-cache` is now [`@apollo/server-plugin-response-cache`](https://www.npmjs.com/package/@apollo/server-plugin-response-cache).
- * `apollo-server-plugin-operation-registry` is now [`@apollo/server-plugin-operation-registry`](https://www.npmjs.com/package/@apollo/server-plugin-operation-registry).
- * `apollo-reporting-protobuf` (an internal implementation detail for the usage reporting plugin) is now [`@apollo/usage-reporting-protobuf`](https://www.npmjs.com/package/@apollo/usage-reporting-protobuf).
+  - `apollo-datasource-rest` is now [`@apollo/datasource-rest`](https://www.npmjs.com/package/@apollo/datasource-rest). The `@apollo/datasource-rest` package removes an explicit dependency on `@apollo/server` and is compatible with both Apollo Server 3 and 4. 
```

Comment:
This is not technically accurate...
- `@apollo/server` is the new package name so certainly it wasn't a dependency in the past
- The thing that's removed is actually several other small package dependencies: `apollo-datasource` (the interface that Apollo Server talks to) and `apollo-server-env` (this is just us migrating to `@apollo/fetcher`.  What the point really is is that the API no longer expects to have a `configure` method called on it by Apollo Server (or by you manually?). But it's not a direct dependency.

Reply:
Oh sorry, I forgot to save the file before pushing up  🤦‍♀️

Hmmm, what if we remove the second sentence here? The more I think about it, with this level of detail we should either mention this in the "Fetching Rest" article directly or just omit it if we don't feel like it's important. How do you feel about that?

Reply:
Yeah I think removing it is fine.